src/vector/index.js

```diff
@@ -382,6 +382,7 @@ async function loadLanguage() {
     }
     try {
         await languageHandler.setLanguage(langs);
+        document.getElementById("mx_PageRoot").setAttribute("lang", languageHandler.getCurrentLanguage());
```

Comment:
Instead of using a special id, you should just use `document.documentElement`. That way you don't need to add any id to the `<html>` element at all.